Content.Client/Chat/Managers/ContentMarkupTagManager.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Linq;
+using Content.Client.Chat.MarkupTags;
+using Content.Shared.Chat;
+using Content.Shared.Chat.ContentMarkupTags;
+
+namespace Content.Client.Chat.Managers;
+
+public sealed class ContentMarkupTagManager : ISharedContentMarkupTagManager
+{
+    public Dictionary<string, IContentMarkupTag> ContentMarkupTagTypes => new IContentMarkupTag[]
+    {
+        new ColorValueContentTag(),
```

Comment:
it would be cool if code of DI would just scan assembly for all implementation of IContentMarkupTag and inject them there, and its easy to do! BUT! i am not sure if assembly-scanning is ok for sandboxing, and 2 - i'm not sure if maintainers are okay with that practice.

Reply:
There is a similar implementation for RobustToolbox's MarkupTagTypes that does it this way, and cites sandboxing as the reason why. So it might be a limitation we have to live with. I agree it's not optimal.

Reply:
well, i guess we are stuck with this. Well it is readable at least!